sync/sync.go

```diff
@@ -384,7 +384,8 @@ func (sync *synchronizer) peerIsInTheCommittee(pid peer.ID) bool {
 	}
 
 	for _, key := range p.ConsensusKeys {
-		if sync.state.IsInCommittee(key.Address()) {
+		// TODO: cache address for the better performance
```

Comment:
This comment can be removed. We cache the address inside the `ValidatorKey` object.